uint8 WHAT_ARE_YOU_DOING=0
uint8 WHY_ARE_YOU_DOING_THIS=1
uint8 WHAT_IS_YOUR_SUBGOAL=2
uint8 HOW_DO_YOU_ACHIEVE_YOUR_SUBGOAL=3
uint8 WHAT_IS_YOUR_GOAL=4
uint8 HOW_DO_YOU_ACHIEVE_YOUR_GOAL=5
uint8 WHAT_WENT_WRONG=6
uint8 WHAT_IS_NEXT_ACTION_IF_SUCCESS=7
uint8 WHAT_IS_NEXT_ACTION_IF_FAIL=8
uint8 WHAT_ARE_CURRENT_PRE_CONDITIONS=9
uint8 WHAT_ARE_CURRENT_POST_CONDITIONS=10
uint8 question
---
string answer